packages/components/app/styles/components/pagination-bar.scss

```diff
@@ -2,4 +2,127 @@
 // PAGINATIONBAR
 //
 
-// .hds-pagination-bar {}
+@use "../mixins/focus-ring" as *;
+
+.hds-pagination-bar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 36px;
+  color: var(--token-color-foreground-primary);
+  font-size: 13px;
+}
+
+// TODO: refactor!!! (Should not reference nested component classes)
```

Comment:
These styles are still very rough and are mainly for getting the rough visual styling in place. (Wasn't sure of best way of styling the `Hds::Form::Select::Field` I used for example so I added kind of hacky styles for now.)